docs/api/createRouteAction.md

```diff
@@ -237,6 +237,44 @@ export function EnrollmentPage() {
 }
 ```
 
+### Creating a Enhanced Form
+
+Although we recommend using [progressively enhanced forms on the server](./createServerAction.md#creating-a-progressively-enhanced-form) whenever possible, one of the benefits of running actions on the client is that we can still create enhanced forms. To accomplish this we need to pass information to our action using form elements like `<input>`. Any data need to be sent that end users don't enter can be added with an `<input>` with `type="hidden"`.
```

Comment:
The title should read "Creating an Enhanced Form"

The last sentence of this paragraph can be better worded. e.g "Any required data that the end user doesn't need to see, can be represented with an `<input>` with `type="hidden"`"